### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by eamador1**
In this project, the following was done:
___
-Use ES6 syntax.✅
-Divide code into modules.✅
-Create a separate file for each module in the modules directory.✅
-Import modules into the index.js file. (Tip: the more modules the better!)✅
-Keep all funcionalities of the app without errors.✅
-Refactor your functions to arrow functions.✅
-Use let and const in the correct way.✅
-Use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)✅
-Download the library into the project using npm.✅
-Import it as a module.✅

### CODE REVIEW

File: modules/spa.js
Comment (LionRouge1) [inline modules/spa.js:1]: - [ ] Nice work implementing the navigation bar :clap: However on the page load no section is display except the the time and the navigation bar. Therefore I suggest you select either the section responsible for displaying the list of added books or the section responsible for adding a new book as a default section that will be shown any time the page loaded.
  Kindly take a look at this short video :point_down:  to have more understanding of the issues
<video src="https://github.com/eamador1/awesomeLibrary-ES6/assets/91142432/c74d7922-8aaa-47e9-826e-27a1b4f0ed71" alt="issues" />

File: README.md
Comment (LionRouge1) [inline README.md:1]: - [ ] Great job using the recommended Microverse README file template :+1:  However your README file look a little messy and difficult to read the content. Kindly make sure you follow the same structure as the recommended [README template](https://github.com/microverseinc/readme-template) file. That will make you project looks more professional and clean.
### This is how your README file looks like :point_down: 
Screenshot from 2023-08-14 23-47-43

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @eamador1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Use ES6 module :heavy_check_mark: 
- Use arrow syntax for functions :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last code review, the following changes were implemented:
___

- Display library when page is loaded
- Structure readme file accordingly

**Review by iambenkis — APPROVED** [review]
Hi @eamador1  , 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

-  There is no linter error ✔️ 
-  Use of best practices of HTML , CSS and Javascript. ✔️ 
- Use of Es6 syntax. ✔️ 
- The project is well set up. ✔️ 
- Use of Luxon packages to get the date. ✔️ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 0xMALVEE** [comment]
Hi @eamador1 ,

🟢 Status: APPROVED 🏆 🎉 🎊
You have done a great job on this project, keep it up 👍 🥇. Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
congratulations! 🎉💯🌟

Top highlight 🎯
There is no linter error ✔️
Use of best practices of HTML , CSS and Javascript. ✔️
Use of Es6 syntax. ✔️
The project is well set up. ✔️
Use of Luxon packages to get the date. ✔️

**Pull request merged**